### ISSUE TITLE
Accessibility node should auto focus on the bottom sheet when showBottomSheet is used

### ISSUE THREAD

**Issue by mehmetf** [flutter/flutter#21028]
<eom>

**Comment by jonahwilliams** [comment]
@mehmetf Did this come up in an internal a11y review?

TalkBack doesn't change focus unless the previous element disappears (usually during a route-like event).  In the case of the bottom sheet, the previous elements are still available. For example, we don't auto focus snackbars either, even if they have an action.

**Comment by mehmetf** [comment]
Most likely it did yes. The bug opened does not have much more details. I will assign to you so you can ask for clarification internally. Thanks Jonah.
